.github/workflows/Test.yml

```diff
@@ -242,6 +242,15 @@ jobs:
         cd scripts/msys2-${{ matrix.pkg }}
         makepkg-mingw --noconfirm --noprogressbar -sCLf
 
+    - name: 'Prepare NSIS build'
```

Comment:
Please, create the NSIS installer in a separated job, *after* `win-test`. See https://github.com/ghdl/ghdl/actions/runs/868960417.

We want to first build GHDL, then test it, and generate the installer only if the tests are successful.